Source/LinqToDB/Sql/Sql.cs

```diff
@@ -668,6 +668,7 @@ void IExtensionCallBuilder.Build(ISqExtensionBuilder builder)
 			{
 				var str = builder.GetExpression("str");
 
+				var whiteSpaces = $"%[^{WHITESPACES}]%";
```

Comment:
Under C#10, interpolated strings with constant parameters can also be constant. (also for other similar lines later in file)

```suggestion
				const string whiteSpaces = $"%[^{WHITESPACES}]%";
```